index.html

```diff
@@ -2,239 +2,320 @@
-   </li>
-
-   <hr class="about-break"/>
+  </section>
+
+  <section class="About-me">
+    <h4 class="about">About me</h4>
+    <p class="about-words">
+      Hello I’m a software developer! I can help you build a product , feature
+      or website Look through some of my work and experience! If you like what
+      you see and have a project you need coded, don’t hestiate to contact me.
+    </p>
 
- </section>
+    <li class="resume-button">
+      <a href="#"><img src="images/resume.png" alt="" /></a>
+    </li>
 
- <section class="myskills">
+    <hr class="about-break" />
+  </section>
 
-   <ul class="language">
-       <a class="icon" href="#"><img src="images/language.png" alt=""/></a>
-       <li>
+  <section class="myskills">
+    <ul class="language">
+      <a class="icon" href="#"><img src="images/language.png" alt="" /></a>
+      <li>
         <a class="tech">Languages</a>
-       </li>
-   </ul>
-   <ul class="list"> 
+      </li>
+    </ul>
+    <ul class="list">
       <li class="list">
         <a class="skillset" href="#">Javascript</a>
         <a class="skillset" href="#">Ruby</a>
         <a class="skillset" href="#">Html</a>
         <a class="skillset" href="#">Css</a>
-      </li>  
-       
-   </ul>
+      </li>
+    </ul>
 
-   <ul class="language">
-       <a class="icon" href="#"><img src="images/frameworks.png" alt=""/></a>
-       <li>
+    <ul class="language">
+      <a class="icon" href="#"><img src="images/frameworks.png" alt="" /></a>
+      <li>
         <a class="tech">Frameworks</a>
-       </li>
-   </ul>
-   <ul class="list">  
+      </li>
+    </ul>
+    <ul class="list">
       <li class="list">
         <a class="skillset" href="#">Bootstrap</a>
         <a class="skillset" href="#">Ruby on Rails</a>
         <a class="skillset" href="#">RSPec</a>
         <a class="skillset" href="#">CapyBara</a>
         <a class="skillset" href="#">Selenium</a>
-      </li>  
-       
-   </ul>
+      </li>
+    </ul>
 
-   <ul class="language">
-       <a class="icon" href="#"><img src="images/skills.png" alt=""/></a>
-       <li>
+    <ul class="language">
+      <a class="icon" href="#"><img src="images/skills.png" alt="" /></a>
+      <li>
         <a class="tech">Skills</a>
-       </li>
-   </ul>
-   <ul class="list">   
+      </li>
+    </ul>
+    <ul class="list">
       <li class="list">
         <a class="skillset" href="#">Codekit</a>
         <a class="skillset" href="#">Github</a>
         <a class="skillset" href="#">Codepen</a>
         <a class="skillset" href="#">Gitlab</a>
         <a class="skillset" href="#">Terminal</a>
       </li>
-       
-   </ul>
-
+    </ul>
+  </section>
 
- </section>
+  <section class="form">
+    <p class="form-words">
+      I'm always interested in hearing about new projects, so if you'd like to
+      chat please get in touch.
+    </p>
 
+    <form
+      action="https://formspree.io/f/{form_id}"
+      method="post"
+      class="contact-form">
```

Comment:
- [ ] You did good work however the form is not working. In order to make it work kindly replace `{form_id}` in the form action will the specific `id` provided after when creating an account on [Formspree](https://formspree.io/html/).